ui2/src/pages/error/ServiceRequired.tsx

```diff
@@ -0,0 +1,259 @@
+import React, { useEffect, useState, useMemo } from 'react';
```

Comment:
This page shares a lot of code with `Landing.tsx`, the actual modal they both use can probably be refactored into one component